src/configuration/decoration.ts

```diff
@@ -1,11 +1,21 @@
 import * as vscode from 'vscode';
+import * as path from 'path';
 import { IConfiguration } from './iconfiguration';
 
 class DecorationImpl {
+  private _mark: vscode.TextEditorDecorationType;
   private _default: vscode.TextEditorDecorationType;
   private _searchHighlight: vscode.TextEditorDecorationType;
   private _easyMotion: vscode.TextEditorDecorationType;
 
+  public set Mark(value: vscode.TextEditorDecorationType) {
```

Comment:
Lowercase for new methods, please. Not sure why there's some inconsistency, I'll go through the code base and clean that up some time.

Reply:
done